fairroot/base/event/FairTimeStamp.h

```diff
@@ -74,13 +74,13 @@ class FairTimeStamp : public FairMultiLinkedData_Interface
         return out;
     }
 
-    virtual bool operator<(const FairTimeStamp* rValue) const { return GetTimeStamp() < rValue->GetTimeStamp(); }
+    bool operator<(const FairTimeStamp& rValue) const { return fTimeStamp < rValue.fTimeStamp; }
 
   protected:
     Double_t fTimeStamp{-1};        //< Time of digit or Hit  [ns]
     Double_t fTimeStampError{-1};   //< Error on time stamp
 
-    ClassDefOverride(FairTimeStamp, 4);
+    ClassDefOverride(FairTimeStamp, 5);
```

Comment:
Why do you need to update the version? Can you please explain?

Reply:
Since there is no change in member variables, I think the version should stay the same.

Reply:
I agree. The version number should stay at 4.